docs/descriptor/packages.rst

```diff
@@ -19,13 +19,35 @@ Packages are defined in the ``install`` subsection.
 ``repositories``
 ----------------
 Cekit uses all repositories configured inside the image. You can also specify additional
-repositories using repositories subsection. Cekit currently supports three ways of defining
+repositories using repositories subsection. Cekit currently supports four ways of defining
 additional repositories:
 
+* Plain
 * RPM
 * ODCS
 * URL
 
+.. note::
+   See :ref:`Repository mangement<repository_management>` to learn about best practices for repository
+   definitions.
+
+.. _repo_plain:
+
+``Plain``
```

Comment:
Probably should not be a "code".

Reply:
everything is as code here

Reply:
Sure, but there is no "Plain" key in contrast to other options.

P.S. `ODCS` should be: `odcs`, `RPM` should be: `rpm`, so everything should be lower case, if you use the "code" formatting. If you don't, then drop code formatting.